### PR TITLE
Add rubygems links

### PR DESCRIPTION
**Pull request by viraptor**
Add direct links to the gem metadata.
This allows better automatic dependency management.

### CODE REVIEW

**Comment by jessewgibbs** [comment]
@waltjones could you review this?

File: rollbar.gemspec [hunk 1]
```diff
@@ -18,4 +18,12 @@ Gem::Specification.new do |gem|
   gem.version       = Rollbar::VERSION
 
   gem.add_runtime_dependency 'multi_json'
+
+  if s.respond_to?(:metadata)
```
Comment (waltjones) [inline rollbar.gemspec]: @viraptor thank you for the PR. `s` isn't defined. Can you change to `gem`?

Comment (viraptor) [reply rollbar.gemspec]: Right, sorry. Silly copy-paste. Fixing now...

**Comment by waltjones** [comment]
@viraptor Looks good. Thank you!

**Pull request merged**